.codeclimate.yml

```diff
@@ -42,7 +42,6 @@ plugins:
     config:
       strings:
       - HACK
```

Comment:
Yes, same thing.  But, it would be nice to indicate code in one way for PR's to make sure it gets essentially blocked by Code Climate because it must be fixed.  I think Hack would be nice to keep. I haven't seen it used.  But, nice to have in the case someone does want to do a little more pseudo code / notes in code before they code.